INSTALL

```diff
@@ -1,37 +1,36 @@
 Installation Instructions
 *************************
 0. Required: 
-    cmake      >= 2.6.2
+    cmake      >= 3.21.0
     ibus       >= 1.3
     glib       >= 2
     gob        >= 2.0.16
     gtk        >= 3
-    libchewing >= 0.3.3
+    libchewing >= 0.5.1
     libX11
    
 1. Remove the old build cache:
+  
   If you have previous ibus-chewing, or change between ibus 1.1 and ibus 1.2,
   You may need to remove CMakeCache.txt
 
 2. Extract source and enter the extracted directory, then:
-cmake . -DCMAKE_INSTALL_PREFIX='/usr' -DLIBEXEC_DIR='/usr/libexec'
-make
 
-Change CMAKE_INSTALL_PREFIX and LIBEXEC_DIR if you want to install
```

Comment:
`LIBEXEC_DIR` is still necessary as it's used in some places: https://github.com/search?q=repo%3Achewing%2Fibus-chewing%20LIBEXEC_DIR&type=code